blueocean-core-js/test/js/fetch-spec.js

```diff
@@ -67,6 +67,7 @@ describe('Fetch', () => {
                             assert.fail(null, null, 'should not call success handler');
                         },
                         error => {
+                            console.log('XXXXX:' ,error);
```

Comment:
Leftover debugging, or is this meant to be here?

Reply:
No cheers